qiskit/providers/ibmq/api/clients/websocket.py

```diff
@@ -114,7 +114,7 @@ def from_bytes(cls, json_string: bytes) -> 'WebsocketResponseMethod':
             parsed_dict = json.loads(json_string.decode('utf8'))
         except (ValueError, AttributeError) as ex:
             exception_to_raise = WebsocketIBMQProtocolError(
-                'Unable to parse the message received from the server: {}'.format(json_string))
+                'Unable to parse the message received from the server: {!r}'.format(json_string))
```

Comment:
Mypy reports the following error when formatting bytes:
```
On Python 3 '{}'.format(b'abc') produces "b'abc'"; use !r if this is a desired behavior
```

Specifying `!r` seemed like the appropriate action to take as a result.